spike_dasm/spike-dasm.cc

```diff
@@ -41,7 +41,11 @@ int main(int argc, char** argv)
     return 1;
   }
 
-  disassembler_t* disassembler = new disassembler_t(xlen);
+  bool zfinx = lowercase.find(std::string("zfinx")) != std::string::npos;
+  bool zdinx = lowercase.find(std::string("zdinx")) != std::string::npos;
+  bool zqinx = lowercase.find(std::string("zqinx")) != std::string::npos;
+
+  disassembler_t* disassembler = new disassembler_t(xlen, zfinx || zdinx || zqinx);
```

Comment:
Can't this use the same ISA string processing as `spike_main`?

Reply:
Yes. However, the ISA parser in spike-dasm is very simple: it only parse the xlen info before. So I don't know whether is it necessary to use the same ISA string processing as spike_main.

Reply:
I think we should move to having unified ISA-string processing, even if it results in minor backwards incompatibilities for spike-dasm.

Reply:
The latter, not the former.  Seeing "unknown" all over the place is not helpful; it's better to disassemble as much as possible.

Reply:
If/when Spike supports conflicting extensions (with overlapping opcodes) then we'd need to have the disassembler consider more of the ISA string. But for now it can simply always support all extensions.